src/cryptography/hazmat/bindings/openssl/ssl.py

```diff
@@ -438,6 +440,13 @@
 long (*SSL_get_tlsext_status_ocsp_resp)(SSL *, const unsigned char **) = NULL;
 #endif
 
+#ifdef SSL_CTRL_SET_TLSEXT_STATUS_REQ_TYPE
```

Comment:
Well, as you mentioned on IRC, ```SSL_set_tlsext_status_type``` was added in OpenSSL 0.9.8h and I checked and ```SSL_CTX_set_tlsext_status_cb``` was also added in 0.9.8h, so I don't think we need to add a separate comment as [this comment](https://github.com/eeshangarg/cryptography/blob/add-openssl-binding-for-SSL_set_tlsext_status_type/src/cryptography/hazmat/bindings/openssl/ssl.py#L325) already says that the four macros following it were added in 0.9.8h. :-)

Thanks a lot for your help and guidance! Let me know if this needs further work!